matcher/service/openreview_interface.py

```diff
@@ -584,6 +584,10 @@ def __init__(
         self.probability_limits = float(
             self.config_note.content.get("randomized_probability_limits", 1.0)
         )
+        self.perturbation = float(
+            self.config_note.content.get("randomized_perturbation", 0.0)
```

Comment:
is 0 ok as default here?

Reply:
Yes. Setting it to 0 simply nullifies its effect.